Source/Core/VideoBackends/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 add_subdirectory(OGL)
 add_subdirectory(Null)
 add_subdirectory(Software)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
@ligfx I can see that because they stopped adding new simple variables for platform detection.
Just the same way they moved away from the CMAKE_COMPILER_IS_GNU<LANG> to detect just GCC in favour of CMAKE_<LANG>_COMPILER_ID